tasks/main.yml

```diff
@@ -10,16 +13,16 @@
 - name: Check if OS is supported
   include_tasks: os-check.yml
 
-- name: Resolve datadog_tracked_checks later to defend against variable presidence issues arising from dynamically included null datadog_checks
+- name: Resolve agent_datadog_tracked_checks later to defend against variable presidence issues arising from dynamically included null agent_datadog_checks
```

Comment:
The `datadog_checks` in this name refers to the configuration variable, not our internal variable. I believe it should remain as is.

Reply:
Again, this is marked as resolved but I don't think it is.